tests/phpunit/tests/avatar.php

```diff
@@ -169,7 +169,7 @@ public function get_avatar_comment_types_filter( $comment_types ) {
 
 	public function test_get_avatar() {
 		$img = get_avatar( 1 );
-		$this->assertSame( preg_match( "|^<img alt='[^']*' src='[^']*' srcset='[^']*' class='[^']*' height='[^']*' width='[^']*' loading='lazy'/>$|", $img ), 1 );
+		$this->assertSame( preg_match( "|^<img alt='[^']*' src='[^']*' srcset='[^']*' class='[^']*' height='[^']*' width='[^']*' loading='lazy' decoding='async'/>$|", $img ), 1 );
```

Comment:
As there are now 2 different assertions in the same test, I'd suggest adding a message to both. Why? To identify which test assertion failed

Reply:
i'm not sure I understand this change, are the tests failing here for you?

Reply:
👍🏼 implemented in patch